### ISSUE TITLE
Problem with gcc-atomic and Cinder Boost binaries

### ISSUE THREAD

**Issue by ryanbartley** [wieden-kennedy/Cinder-WebSocketPP#5]
Original issue opened on Cinder [here](https://github.com/cinder/Cinder/issues/560). I'm using glNext and keep up the Cinder version of SpaceBrew [here](https://github.com/Spacebrew/ciSpaceBrew). It looks like everything was recently updated but I'm still getting the same gcc-atomic problems and I'm not sure where to look.

**Comment by gaborpapp** [comment]
This is an OS X boost 1.55 issue as far as I know. If you build in 32-bit it works.

**Comment by ryanbartley** [comment]
Build Cinder Osx in 32-bit?

**Comment by gaborpapp** [comment]
Yes, Cinder, your app, and also boost libs need to be 32-bit, Boost 1.55 has this atomic issue in 64-bit, which was fixed in later versions. If you don't need 64-bit it is probably easier to switch to 32-bit instead of replacing boost.
